autoconfigure/storage-elasticsearch-aws/src/main/java/zipkin/autoconfigure/storage/elasticsearch/aws/ElasticsearchDomainEndpoint.java

```diff
@@ -13,73 +13,79 @@
+import com.linecorp.armeria.common.HttpStatusClass;
+import io.netty.util.AttributeKey;
 import java.io.IOException;
-import java.util.Collections;
-import java.util.List;
-import java.util.logging.Logger;
-import okhttp3.HttpUrl;
-import okhttp3.OkHttpClient;
-import okhttp3.Request;
-import okhttp3.Response;
-import okio.Buffer;
-import zipkin2.elasticsearch.ElasticsearchStorage;
+import java.util.concurrent.CompletionException;
+import java.util.function.Function;
+import java.util.function.Supplier;
 
-import static zipkin2.elasticsearch.internal.JsonReaders.enterPath;
+import static zipkin.autoconfigure.storage.elasticsearch.aws.ZipkinElasticsearchAwsStorageAutoConfiguration.JSON;
 
-final class ElasticsearchDomainEndpoint implements ElasticsearchStorage.HostsSupplier {
-  static final Logger log = Logger.getLogger(ElasticsearchDomainEndpoint.class.getName());
+final class ElasticsearchDomainEndpoint implements Supplier<EndpointGroup> {
+  static final AttributeKey<String> NAME = AttributeKey.valueOf("name");
 
-  final OkHttpClient client;
-  final Request describeElasticsearchDomain;
+  final Function<Endpoint, HttpClient> clientFactory;
+  final Endpoint endpoint;
+  final AggregatedHttpRequest describeElasticsearchDomain;
 
-  ElasticsearchDomainEndpoint(OkHttpClient client, HttpUrl baseUrl, String domain) {
-    if (client == null) throw new NullPointerException("client == null");
-    if (baseUrl == null) throw new NullPointerException("baseUrl == null");
-    if (domain == null) throw new NullPointerException("domain == null");
-    this.client = client;
+  ElasticsearchDomainEndpoint(Function<Endpoint, HttpClient> clientFactory, Endpoint endpoint,
+      String domain) {
+    this.clientFactory = clientFactory;
+    this.endpoint = endpoint;
     this.describeElasticsearchDomain =
-        new Request.Builder()
-            .url(baseUrl.newBuilder("2015-01-01/es/domain").addPathSegment(domain).build())
-            .tag("get-es-domain")
-            .build();
+        AggregatedHttpRequest.of(HttpMethod.GET, "/2015-01-01/es/domain/" + domain);
   }
 
-  @Override
-  public List<String> get() {
-    try (Response response = client.newCall(describeElasticsearchDomain).execute()) {
-      String body = response.body().string();
-      if (!response.isSuccessful()) {
-        String message =
-            describeElasticsearchDomain.url().encodedPath()
-                + " failed with status "
-                + response.code();
-        if (!body.isEmpty()) message += ": " + body;
-        throw new IllegalStateException(message);
-      }
-      JsonReader endpointReader = JsonReader.of(new Buffer().writeUtf8(body));
-      endpointReader = enterPath(endpointReader, "DomainStatus", "Endpoints");
-      if (endpointReader != null) endpointReader = enterPath(endpointReader, "vpc");
+  @Override public DnsAddressEndpointGroup get() {
+    // We want "string = http.client.get() labeling the request NAME as "es-get-domain"
```

Comment:
patience is waning.. there's still a distraction in the logs. also it is hard for me to tell if the :authority and "host" are redundant or not. Before, we were logging http 1.1 semantic request headers and no longer doing that so I'm just guessing.. I wish we could tell what was on the wire, which was the point of the whole logging thing..


```
2019-07-30 19:51:30.709  INFO 6700 --- [-worker-nio-2-2] c.l.a.c.l.LoggingClient                  : Request: {startTime=2019-07-30T11:51:30.702Z(1564487490702871), length=0B, duration=4060µs(4060249ns), scheme=none+h1, headers=[:method=GET, :path=/2015-01-01/es/domain/zipkin, :authority=es.ap-southeast-1.amazonaws.com, :scheme=https, x-amz-date=20190730T115130Z, host=es.ap-southeast-1.amazonaws.com, authorization=AWS4-HMAC-SHA256 Credential=AKIAJGY6SK4B32OMPXUQ/20190730/ap-southeast-1/es/aws4_request, SignedHeaders=host;x-amz-date, Signature=ca1b105e97eea5fbf55db6686116a4fdf4485b4350d06fc4330db415cf372c50, b3=351c9cc6425033df-f911e9cad5725443-1-351c9cc6425033df, accept-encoding=gzip,deflate, user-agent=armeria/0.89.0]}
2019-07-30 19:51:30.764  INFO 6700 --- [-worker-nio-2-2] c.l.a.c.l.LoggingClient                  : Response: {startTime=2019-07-30T11:51:30.749Z(1564487490749563), length=192B, duration=14894µs(14894747ns), headers=[:status=403, x-amzn-requestid=5ee7e017-b2c0-11e9-b7e1-d3df7696a475, x-amzn-errortype=InvalidSignatureException, content-type=application/json, content-length=192, date=Tue, 30 Jul 2019 11:51:30 GMT]}
2019-07-30 19:51:30.764  WARN 6700 --- [-worker-nio-2-1] z.s.i.BodyIsExceptionMessage             : Unexpected error handling request.

java.lang.RuntimeException: couldn't lookup AWS ES domain endpoint
	at zipkin.autoconfigure.storage.elasticsearch.aws.ElasticsearchDomainEndpoint.get(ElasticsearchDomainEndpoint.java:64) ~[elasticsearch-aws.jar!/:?]
	at zipkin.autoconfigure.storage.elasticsearch.aws.ElasticsearchDomainEndpoint.get(ElasticsearchDomainEndpoint.java:34) ~[elasticsearch-aws.jar!/:?]
	at zipkin2.server.internal.elasticsearch.LazyHttpClientImpl.getEndpoint(LazyHttpClientImpl.java:64) ~[classes!/:?]
	at zipkin2.server.internal.elasticsearch.LazyHttpClientImpl.get(LazyHttpClientImpl.java:56) ~[classes!/:?]
	at zipkin2.elasticsearch.ElasticsearchStorage.http(ElasticsearchStorage.java:298) ~[zipkin-storage-elasticsearch-2.15.1-SNAPSHOT.jar!/:?]
	at zipkin2.elasticsearch.AutoValue_ElasticsearchStorage.http(AutoValue_ElasticsearchStorage.java:44) ~[zipkin-storage-elasticsearch-2.15.1-SNAPSHOT.jar!/:?]
	at zipkin2.elasticsearch.VersionSpecificTemplates.get(VersionSpecificTemplates.java:188) ~[zipkin-storage-elasticsearch-2.15.1-SNAPSHOT.jar!/:?]
	at zipkin2.elasticsearch.ElasticsearchStorage.ensureIndexTemplates(ElasticsearchStorage.java:268) ~[zipkin-storage-elasticsearch-2.15.1-SNAPSHOT.jar!/:?]
	at zipkin2.elasticsearch.AutoValue_ElasticsearchStorage.ensureIndexTemplates(AutoValue_ElasticsearchStorage.java:29) ~[zipkin-storage-elasticsearch-2.15.1-SNAPSHOT.jar!/:?]
	at zipkin2.elasticsearch.ElasticsearchStorage.spanConsumer(ElasticsearchStorage.java:221) ~[zipkin-storage-elasticsearch-2.15.1-SNAPSHOT.jar!/:?]
	at zipkin2.server.internal.brave.TracingStorageComponent.spanConsumer(TracingStorageComponent.java:49) ~[classes!/:?]
	at zipkin2.collector.Collector.store(Collector.java:178) ~[zipkin-collector-2.15.1-SNAPSHOT.jar!/:?]
	at zipkin2.collector.Collector.accept(Collector.java:124) ~[zipkin-collector-2.15.1-SNAPSHOT.jar!/:?]
	at zipkin2.server.internal.ZipkinHttpCollector.lambda$validateAndStoreSpans$0(ZipkinHttpCollector.java:166) ~[classes!/:?]
	at com.linecorp.armeria.common.AbstractRequestContext.lambda$makeContextAware$1(AbstractRequestContext.java:69) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.AbstractRequestContext.lambda$makeContextAware$1(AbstractRequestContext.java:69) ~[armeria-0.89.0.jar!/:?]
	at io.micrometer.core.instrument.AbstractTimer.record(AbstractTimer.java:160) ~[micrometer-core-1.2.0.jar!/:1.2.0]
	at io.micrometer.core.instrument.Timer.lambda$wrap$0(Timer.java:144) ~[micrometer-core-1.2.0.jar!/:1.2.0]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.38.Final.jar!/:4.1.38.Final]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.RuntimeException: /2015-01-01/es/domain/zipkin failed: The request signature we calculated does not match the signature you provided. Check your AWS Secret Access Key and signing method. Consult the service documentation for details.
	at zipkin.autoconfigure.storage.elasticsearch.aws.AWSSignatureVersion4.lambda$execute$3(AWSSignatureVersion4.java:125) ~[elasticsearch-aws.jar!/:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:642) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2073) ~[?:?]
	at com.linecorp.armeria.common.HttpMessageAggregator.onComplete(HttpMessageAggregator.java:98) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.stream.FilteredStreamMessage$FilteringSubscriber.onComplete(FilteredStreamMessage.java:290) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.stream.AbstractStreamMessage$CloseEvent.notifySubscriber(AbstractStreamMessage.java:363) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.stream.DefaultStreamMessage.notifySubscriberOfCloseEvent(DefaultStreamMessage.java:194) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.stream.DefaultStreamMessage.handleCloseEvent(DefaultStreamMessage.java:363) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.stream.DefaultStreamMessage.notifySubscriber0(DefaultStreamMessage.java:306) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.stream.DefaultStreamMessage.notifySubscriber(DefaultStreamMessage.java:253) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.stream.DefaultStreamMessage.addObjectOrEvent(DefaultStreamMessage.java:239) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.stream.DefaultStreamMessage.close(DefaultStreamMessage.java:369) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.client.HttpResponseDecoder$HttpResponseWrapper.closeAction(HttpResponseDecoder.java:296) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.client.HttpResponseDecoder$HttpResponseWrapper.close(HttpResponseDecoder.java:279) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.client.HttpResponseDecoder$HttpResponseWrapper.close(HttpResponseDecoder.java:267) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.client.Http1ResponseDecoder.channelRead(Http1ResponseDecoder.java:212) ~[armeria-0.89.0.jar!/:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:438) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:328) ~[netty-codec-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:302) ~[netty-codec-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:253) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.handler.logging.LoggingHandler.channelRead(LoggingHandler.java:241) ~[netty-handler-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1475) ~[netty-handler-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.handler.ssl.SslHandler.decodeNonJdkCompatible(SslHandler.java:1236) ~[netty-handler-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1273) ~[netty-handler-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:505) ~[netty-codec-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:444) ~[netty-codec-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:283) ~[netty-codec-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.handler.flush.FlushConsolidationHandler.channelRead(FlushConsolidationHandler.java:154) ~[netty-handler-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:287) ~[netty-handler-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1421) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:930) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:163) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:697) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:632) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:549) ~[netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:511) [netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:918) [netty-common-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.38.Final.jar!/:4.1.38.Final]
	... 2 more

2019-07-30 19:51:31.504  WARN 6700 --- [-worker-nio-2-2] .a.c.b.RequestContextCurrentTraceContext : Attempted to propagate trace context, but no request context available. Did you forget to use RequestContext.contextAwareExecutor() or RequestContext.makeContextAware()?

com.linecorp.armeria.common.brave.RequestContextCurrentTraceContext$LogRequestContextWarningOnce$NoRequestContextException: null
	at com.linecorp.armeria.common.brave.RequestContextCurrentTraceContext$LogRequestContextWarningOnce$ClassLoaderHack.<clinit>(RequestContextCurrentTraceContext.java:266) ~[armeria-brave-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.brave.RequestContextCurrentTraceContext$LogRequestContextWarningOnce.get(RequestContextCurrentTraceContext.java:254) ~[armeria-brave-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.brave.RequestContextCurrentTraceContext$LogRequestContextWarningOnce.get(RequestContextCurrentTraceContext.java:247) ~[armeria-brave-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.RequestContext.mapCurrent(RequestContext.java:108) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.brave.RequestContextCurrentTraceContext.getRequestContextOrWarnOnce(RequestContextCurrentTraceContext.java:232) ~[armeria-brave-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.brave.RequestContextCurrentTraceContext.get(RequestContextCurrentTraceContext.java:143) ~[armeria-brave-0.89.0.jar!/:?]
	at brave.Tracer.nextSpan(Tracer.java:472) ~[brave-5.6.9.jar!/:?]
	at brave.http.HttpClientHandler.nextSpan(HttpClientHandler.java:127) ~[brave-instrumentation-http-5.6.9.jar!/:?]
	at brave.http.HttpClientHandler.handleSend(HttpClientHandler.java:89) ~[brave-instrumentation-http-5.6.9.jar!/:?]
	at com.linecorp.armeria.client.brave.BraveClient.execute(BraveClient.java:121) ~[armeria-brave-0.89.0.jar!/:?]
	at com.linecorp.armeria.client.brave.BraveClient.execute(BraveClient.java:56) ~[armeria-brave-0.89.0.jar!/:?]
	at zipkin.autoconfigure.storage.elasticsearch.aws.AWSSignatureVersion4.lambda$execute$2(AWSSignatureVersion4.java:99) ~[elasticsearch-aws.jar!/:?]
	at java.util.concurrent.CompletableFuture.uniComposeStage(CompletableFuture.java:1106) ~[?:?]
	at java.util.concurrent.CompletableFuture.thenCompose(CompletableFuture.java:2235) ~[?:?]
	at zipkin.autoconfigure.storage.elasticsearch.aws.AWSSignatureVersion4.execute(AWSSignatureVersion4.java:96) ~[elasticsearch-aws.jar!/:?]
	at zipkin.autoconfigure.storage.elasticsearch.aws.AWSSignatureVersion4.execute(AWSSignatureVersion4.java:52) ~[elasticsearch-aws.jar!/:?]
	at com.linecorp.armeria.client.logging.LoggingClient.execute(LoggingClient.java:172) ~[armeria-0.89.0.jar!/:?]
	at zipkin2.server.internal.elasticsearch.RawContentLoggingClient.lambda$execute$0(RawContentLoggingClient.java:47) ~[classes!/:?]
	at java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1072) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2073) ~[?:?]
	at com.linecorp.armeria.common.HttpMessageAggregator.onComplete(HttpMessageAggregator.java:98) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.stream.AbstractStreamMessage$CloseEvent.notifySubscriber(AbstractStreamMessage.java:363) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.stream.FixedStreamMessage.notifySubscriberOfCloseEvent(FixedStreamMessage.java:126) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.stream.EmptyFixedStreamMessage.doRequest(EmptyFixedStreamMessage.java:32) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.stream.FixedStreamMessage.request(FixedStreamMessage.java:88) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.stream.AbstractStreamMessage$SubscriptionImpl.request(AbstractStreamMessage.java:309) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.HttpMessageAggregator.onSubscribe(HttpMessageAggregator.java:55) ~[armeria-0.89.0.jar!/:?]
	at com.linecorp.armeria.common.stream.FixedStreamMessage.lambda$subscribe$1(FixedStreamMessage.java:117) ~[armeria-0.89.0.jar!/:?]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163) [netty-common-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:416) [netty-common-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:515) [netty-transport-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:918) [netty-common-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.38.Final.jar!/:4.1.38.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.38.Final.jar!/:4.1.38.Final]
	at java.lang.Thread.run(Thread.java:834) [?:?]
```